.github/workflows/ci.yml

```diff
@@ -317,7 +317,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        downlevel_release: [1.0.0]
+        downlevel_release: [1.0.0, 1.0.1]
```

Comment:
For MsQuic, we only test latest patch for each minor version to prevent explosion. Additionally, we only really support the latest patch version.

Reply:
Yup I was wondering what "support" meant, and that's a more reasonable definition.